src/index.html

```diff
@@ -20,8 +20,340 @@
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="laptopsComputers"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#tablets"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#photo"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <article
+        class="product-card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="iMac"
+          class="product-card__image"
+        />
+        <h2 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <span class="product-card__code">Product code: 195434</span>
+        <div class="product-card__rating">
+          <ul class="stars stars--4">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+          <div class="product-card__reviews">Reviews: 5</div>
+        </div>
+        <p class="product-card__price">
+          <span class="product-card__label">Price:</span>
+          <span class="product-card__value">$2,199</span>
+        </p>
+        <a
+          href="#"
+          class="product-card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </article>
+      <article class="product-card">
```

Comment:
Add the blank line between the cards
```suggestion
      </article>
      
      <article class="product-card">
```